pkg/apis/ceph.rook.io/v1/cleanup.go

```diff
@@ -17,6 +17,27 @@ limitations under the License.
 package v1
 
 const (
+	// SanitizeDataSourceZero uses /dev/zero as sanitize source
+	SanitizeDataSourceZero SanitizeDataSourceProperty = "zero"
+
+	// SanitizeDataSourceRandom uses /dev/random as entropy source
+	SanitizeDataSourceRandom SanitizeDataSourceProperty = "random"
```

Comment:
The more common pattern in rook is to define consts with the code instead of in the v1 types package

Reply:
I agree but since they are typed I thought here was a good place, also we have other constants following the same pattern in the v1 package.